js/src/util/index.js

```diff
@@ -26,7 +26,7 @@ const toType = obj => {
 
 const getUID = prefix => {
   do {
-    prefix += ~~(Math.random() * MAX_UID) // "~~" acts like a faster Math.floor() here
+    prefix += Math.floor((Math.random() * MAX_UID))
```

Comment:
We don't need that extra brackets, do we?

Reply:
True, that's why the PR is a draft also 😇